### PR TITLE
Start circular animation from previous value

### PR DESCRIPTION
**Pull request by sudoDeznit**

### CODE REVIEW

File: screens/components/AnimatedCircle.js [hunk 1]
```diff
@@ -36,67 +34,48 @@ const AnimatedCircle = ({
...
-            // easing: Easing.out(Easing.ease)
-        }).start();
-    };
-    // animation fn for text value
-    const textAnimation = toValue => {
-        return Animated.timing(textAnimated, {
-            delay: delay,
-            toValue,
-            duration,
+            duration: duration,
```
Comment (xlcrr) [inline screens/components/AnimatedCircle.js:37]: 1 var is enough

**Review by xlcrr — COMMENTED** [review]
Need to test this

**Comment by xlcrr** [comment]
Error on UserStats screen

```
This error is located at:
    in AnimatedCircle (at ProgressCircleCard.js:24)

Error: AnimatedValue: Attempting to set value to undefined
```

simulator_screenshot_41B23A91-7685-42CB-8D04-230083A6D0BA

```
[Fri Sep 10 2021 21:45:52.325]  WARN     Sending `onAnimatedValueUpdate` with no listeners registered.
[Fri Sep 10 2021 21:45:54.337]  WARN     Possible Unhandled Promise Rejection (id: 0):
Error: AnimatedValue: Attempting to set value to undefined
```

**Review by xlcrr — CHANGES_REQUESTED** [review]
bug found

**Comment by xlcrr** [comment]
@sudoDeznit I got a bug on this PR, can you try to duplicate it?

**Comment by sudoDeznit** [comment]
Not able to reproduce the error. Tried old account and also new account with 0 images uploaded.
Were you trying on staging or prod ??

**Comment by xlcrr** [comment]
Does not work. App crashes when you change screen while animation is occuring

**Comment by xlcrr** [comment]
simulator_screenshot_5C6E648C-22CB-4C3A-9C84-E1F598575BCE

**Review by xlcrr — CHANGES_REQUESTED** [review]
Breaks when changing screen while progress animation is happening

**Pull request closed without merging**